types/index.d.ts

```diff
@@ -1975,11 +1975,11 @@ export declare namespace Knex {
     index(
       columnNames: string | readonly (string | Raw)[],
       indexName?: string,
-      indexType?: string
+      options?: Readonly<{indexType?: string, storageEngineIndexType?: storageEngineIndexType, predicate?: QueryBuilder}>
```

Comment:
fyi this change was breaking for TS users as the third parameter no longer accepts plain string (the code still supports it, just a typings issue)

Reply:
Thanks for the report.
You'r right, I can make an hotfix today (at the end of day) if necessary. @kibertoad

Reply:
@OlivierCavadenti Would be great, could you also backport it to 0.95? I'd make a release today then.

Reply:
@B4nan @OlivierCavadenti Fix was published in 0.95.13

@OlivierCavadenti, thank you for the rapid fix!